app/redis-provider/redisProvider.js

```diff
@@ -0,0 +1,27 @@
+class RedisProvider {
+    constructor(client) {
+        this.client = client;
+    }
+
+    async set(key, value) {
+        await this.client.set(key, JSON.stringify(value), 'EX', 3600);
+    }
+
+    async get(key) {
+        return JSON.parse(await this.client.get(key));
+    }
+
+    async exists(key) {
+        return await this.client.exists(key);
+    }
+
+    async clear() {
+        await this.client.FLUSHALL();
```

Comment:
We should investigate if we really need to 'flushall` or `flushbd` could be better